ArduCopter/mode_rtl.cpp

```diff
@@ -65,6 +75,11 @@ void Copter::ModeRTL::run(bool disarm_on_land)
     // call the correct run function
     switch (_state) {
 
+    case RTL_Starting:
+        // should not be reached:
+        _state = RTL_InitialClimb;
+        // FALLTHROUGH
```

Comment:
Wrong fallthrough, should be a non comment `FALLTHROUGH;`

Reply:
Fixed.